### PR TITLE
Update sample.Jenkinsfile

### PR DESCRIPTION
**Pull request by thuliomattheus**

### CODE REVIEW

File: sample.Jenkinsfile [hunk 1]
```diff
@@ -45,7 +45,7 @@ pipeline {
                 }
             }
         }
-        stage("Quality gate") {
+        stage("Quality Gate") {
```
Comment (eric-faria) [inline sample.Jenkinsfile:48]: Isn't better use only non capitalized letters here ?

Comment (eric-faria) [inline sample.Jenkinsfile:48]: how many stages on this object ?

Comment (thuliomattheus) [reply sample.Jenkinsfile:48]: No.

Comment (thuliomattheus) [reply sample.Jenkinsfile:48]: 1

Comment (eric-faria) [reply sample.Jenkinsfile:48]: ok

Comment (eric-faria) [inline sample.Jenkinsfile:48]: is that hard ?

Comment (eric-faria) [inline sample.Jenkinsfile:48]: it looks hard ?

**Review by eric-faria — CHANGES_REQUESTED** [review]

**Pull request merged**